src/components/Dropdown/DropdownMenuButton/DropdownMenuButton.tsx

```diff
@@ -68,6 +71,101 @@ export const DropdownMenuButton: FC<Props & ElementProps> = ({
     [onlyIconTrigger],
   )
 
+  const handleKeyDown = useCallback((e: KeyboardEvent) => {
+    if (!containerRef.current || !document.activeElement) {
+      return
+    }
+
+    const allItems = Array.from(containerRef.current.querySelectorAll("li > *"))
+    const {
+        hoveredItem,
+        enabledItems,
+        focusedIndex
+    } = allItems.reduce((acc: {
+        hoveredItem: Element | null;
+        enabledItems: Element[];
```

Comment:
IMO： tabbable の方がより明示的かも?

```suggestion
        tabbableItems: Element[];
```

Reply:
確かに！tabbableのほうが明示的だと思いましたので、採用させていただきます！